docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -436,7 +436,22 @@ The connector supports pushdown for a number of operations:
 
 .. include:: join-pushdown-enabled-true.fragment
 
-.. include:: no-pushdown-text-type.fragment
+Predicate pushdown support
+""""""""""""""""""""""""""
+
+The connector supports pushdown of predicates on ``VARCHAR`` and ``NVARCHAR``
+columns if the underlying columns in SQL Server use a case-sensitive `collation
+<https://learn.microsoft.com/en-us/sql/relational-databases/collations/collation-and-unicode-support?view=sql-server-ver16>`_.
+
+The following operators are pushed down:
+
+- ``=``
+- ``<>``
+- ``IN``
+- ``NOT IN``
+
+Operators are not pushed down and correct results are not preserved for columns
+using a case-insensitive collation.
```

Comment:
```suggestion
No operators are pushed down for columns using
a case-insensitive collation to ensure correct results.
```

Reply:
I feel like "Operators are not pushed down" reads better than "No operators are pushed down" without changing the meaning.